README.md

```diff
@@ -68,7 +68,7 @@ If there are changes made in the Original repository, you can sync the fork to k
 ## Requirements
 * [ArcGIS Runtime SDK for .NET 100.9 or higher](https://developers.arcgis.com/net/latest/)
 * [XCode 11 or higher](https://developer.apple.com/xcode/downloads/)
-    * App supports iOS 12 and above, but requires the iOS 13 SDK to build. Because ArcGIS Runtime uses [Metal](https://developer.apple.com/metal/), you will not be able to test the app on iOS 12 simulator.
+    * App supports iOS 12 and above, but requires the iOS 13 SDK to build. Because ArcGIS Runtime uses [Metal](https://developer.apple.com/metal/), you will not be able to test the app on iOS 12 simulator or on any simulator on versions of macOS prior to Catalina.
```

Comment:
I would suggest mentioning what is required of the user rather than what isn't supported. Perhaps something like,

```suggestion
    * App supports a minimum deployment target of iOS 12 but requires the iOS 13 SDK to build. Note, because ArcGIS Runtime uses [Metal](https://developer.apple.com/metal/), if you want to simulate the app using an iOS simulator you must be running, at minimum, macOS Catalina and iOS 13.
```